Lib/idlelib/colorizer.py

```diff
@@ -357,7 +357,7 @@ def removecolors(self):
 
 def _color_delegator(parent):  # htest #
     from tkinter import Toplevel, Text
-    from idlelib.idle_test.test_colorizer import source
+    from test.test_idle.test_colorizer import source
```

Comment:
This would introduce a bug if IDLE were installed and tests were not (which is normal on macOS) and this were run.  My thought is to move the (illegal) sample code into a file in idle_test.